lca8s8-d1e415/cod-dlc9qq_lca8s8-d1e415.xml

```diff
@@ -0,0 +1,317 @@
+          speratur vel malum venturum
+          <lb ed="#P"/>impeditur, quod si non faceret: certe pecca<lb ed="#P" break="no"/>ret
+          mortaliter.
+        </p>
+        <p>
+          Haec maxime probantur <ref>50.
+          <lb ed="#P"/>dist. vt constitueretur</ref>. et <ref>c. domino sancto</ref>.
+          <lb ed="#P"/>Et in <ref>c. at si clerici in § de adultrerijs de
+          <lb ed="#P"/>judicis</ref>. et in <ref>c. consilium. de obseruatione
+          <lb ed="#P"/>jeiuniorum</ref>. et in <ref>c. denique. 4. distinctio</ref>. Et in
+          <lb ed="#P"/>alijs partibus. Et de permissione latissi<lb ed="#P" break="no"/>me
+          disserit <ref><name>Joanes Andre.</name> in regula pecca<lb ed="#P" break="no"/>tum.
+          de regulis iuris, q. 3</ref>. Et nominatim
+          <lb ed="#P"/>hanc nostram sententiam loquendo de<lb ed="#P" break="no"/>
+          bello non inferendo indis: probat doctis<lb ed="#P" break="no"/>simus
+            magister <ref><name>Victorien.</name> in 2. relectione <title>de indis</title></ref>.
+        </p>
+        <p>
+          ¶ 2º. principaliter pro<pb ed="#P" n="131-r"/><lb ed="#P" break="no"/>batur
+          prefata conclusio sic: preceptum ne<lb ed="#P" break="no"/>gatiuum
+          habemus quod omni casu est
+          <lb ed="#P"/>seruandum: neque vllo modo vel loco illud
+          <lb ed="#P"/>transgredi licet scilicet <quote source="http://scta.info/resource/ex20_13@1-2">non occides</quote>. Id
+          <lb ed="#P"/>autem legitur <ref>Exodi 20 c.</ref>. Rursus <ref>eius
+          <lb ed="#P"/>libri c. 23</ref>. habetur aliud strictius: <quote source="http://scta.info/resource/ex23_7@3-10">in<lb ed="#P" break="no"/>sontem
+          et justum (inquit, deus) non occides quia auersor impium</quote>. Idem <ref target="http://scta.info/resource/dan13_53@11-15">Danie<lb ed="#P" break="no"/>lis
+            13</ref>. Hinc <ref><name>S. Thomas</name> in 3. senten. dist. 47. arº. 3. ad 1m</ref>. <quote>Innocentem
+          <lb ed="#P"/>bominem passioni et morti tradere con<lb ed="#P" break="no"/>tra
+          eius voluntatem est impium et cru<lb ed="#P" break="no"/>dele</quote>.
+          Haec ille.
+        </p>
+        <p>
+          Legibus namque quas in<lb ed="#P" break="no"/>uiolabiliter
+          ac stricte Deus volebat ob<lb ed="#P" break="no"/>seruari
+          addebat minas in transgressores:
+          <lb ed="#P"/>quod vehemens indicium est: dominum
+          <lb ed="#P"/>velle illa exactissime obseruari juxta il<lb ed="#P" break="no"/>lud:
+          <quote source="http://scta.info/resource/ex22_22@1-5"><!--quote goes over several verses, how to cite?-->viduae non nocebis et pupillo: si
+          <lb ed="#P"/>leseritis eos vociferabuntur ad me: et ego
```

Comment:
Right now, you need to list each verse separated by white space. 

So if the passages extends 5 verses then it would look like this. There are some ideas I have allow the indication of  "ranges" but its not high priority and this works for the moment.

`<quote 
     source="http://scta.info/resource/ex22_22@1-5 
     http://scta.info/resource/ex22_23 
     http://scta.info/resource/ex22_24 
     http://scta.info/resource/ex22_25 
     http://scta.info/resource/ex22_26 
     http://scta.info/resource/ex22_27">`